test/integration/requests/ledger.ts

```diff
@@ -0,0 +1,102 @@
+import { assert } from 'chai'
+import _ from 'lodash'
+
+import {
+  LedgerRequest,
+  LedgerResponse,
+  LedgerClosedRequest,
+  LedgerClosedResponse,
+  LedgerCurrentResponse,
+  LedgerCurrentRequest,
+  LedgerDataRequest,
+  LedgerDataResponse,
+  LedgerEntryRequest,
+  LedgerEntryResponse,
+} from 'xrpl-local'
+
+import serverUrl from '../serverUrl'
+import { setupClient, suiteClientSetup, teardownClient } from '../setup'
+import { verifySuccessfulResponse } from '../utils'
+
+// how long before each test case times out
+const TIMEOUT = 20000
+
+describe('Ledger Methods', function () {
+  this.timeout(TIMEOUT)
+
+  before(suiteClientSetup)
+  beforeEach(_.partial(setupClient, serverUrl))
+  afterEach(teardownClient)
+
+  it('Ledger', async function () {
+    const ledgerRequest: LedgerRequest = {
+      command: 'ledger',
+      ledger_index: 'validated',
+    }
+
+    const ledgerResponse: LedgerResponse = await this.client.request(
+      ledgerRequest,
+    )
+
+    verifySuccessfulResponse(ledgerResponse)
+    assert.ok(ledgerResponse.result.validated)
+  })
+
+  it('LedgerClosed', async function () {
```

Comment:
Should each of these be in a separate file? (Or does it make sense to group them like this?) @mvadari

Reply:
They should be in separate folders. The `test/integration/requests` folder should parallel the `src/models/requests` folder.